netrd/distance/communicability_jsd.py

```diff
@@ -0,0 +1,79 @@
+"""
+communicability_jsd.py
+--------------------------
+
+Distance measure based on the Jensen-Shannon Divergence 
+between the communicability sequence of two graphs as 
+defined in:
+
+Chen, D., Shi, D. D., Qin, M., Xu, S. M., & Pan, G. J. (2018). 
+Complex network comparison based on communicability 
+sequence entropy. Physical Review E, 98(1), 012319.
+
+https://journals.aps.org/pre/abstract/10.1103/PhysRevE.98.012319
+
+author: Brennan Klein
+email: brennanjamesklein@gmail.com
+Submitted as part of the 2019 NetSI Collabathon.
+
+"""
+
+import networkx as nx
+import numpy as np
+import scipy as sp
+from .base import BaseDistance
+
+class CommunicabilityJSD(BaseDistance):
+    def dist(self, G1, G2):
+        """        
```

Comment:
Please add more documentation here. What does the distance do, what is communacibility (briefly), what is stored in `self.results`, what kinds of graphs does this work on.